dataproc/requirements.txt

```diff
@@ -1,4 +1,4 @@
-grpcio>=1.2.0
+grpcio==1.26.0
```

Comment:
@bradmiro any strong feelings about this?

Reply:
No cause for concern AFAIK